src/SpectatorView.Unity/Assets/SpectatorView/Scripts/Compositor/CompositionManager.cs

```diff
@@ -504,7 +526,7 @@ private void Update()
 
             //Set our current frame towards the latest captured frame. Dont get too close to it, and dont fall too far behind 
             int step = (captureFrameIndex - CurrentCompositeFrame);
-            if (step < 8)
+            if (step < (LowLatencyMode ? 2 : 8))

```

Comment:
>2 [](start = 41, length = 1)

nit: I know that these line values weren't your doing, but would it be possible to get these values defined as constants?